pdf_text.go

```diff
@@ -56,7 +56,14 @@ func ConvertPDFText(path string) (BodyResult, MetaResult, error) {
 
 	br := make(chan BodyResult, 1)
 	go func() {
-		body, err := exec.Command("pdftotext", "-q", "-nopgbrk", "-enc", "UTF-8", "-eol", "unix", path, "-").Output()
+		var body []byte
+		var err error	
+		if readability {
+			body, err = exec.Command("pdftotext", "-q", "-nopgbrk", "-enc", "UTF-8", "-eol", "unix", "-layout", path, "-").Output()
```

Comment:
We'd want to collapse the shared arguments into one location and pass in `-layout` if readability is true.